src/previewSupport/handleConsoleLogs.js

```diff
@@ -0,0 +1,32 @@
+import channel from './channel';
+
+function notifyChannel(args) {
+  if (args.length > 0) {
+    const output = args.map(arg => JSON.stringify(arg)).join(' ');
```

Comment:
As discussed IRL, let’s send the values as an array and then stringify on the other end.